tasks/libs/ciproviders/gitlab_api.py

```diff
@@ -1276,6 +1274,7 @@ def update_test_infra_def(file_path, image_tag, is_dev_image=False):
         except yaml.YAMLError as e:
             raise Exit(f"Error while loading {file_path}: {e}") from e
     with open(file_path, "w") as test_infra_version_file:
+        test_infra_version_file.write(prefix_comment)
```

Comment:
```suggestion
        test_infra_version_file.write(prefix_comment + ('\n\n' if prefix_comment else ''))
```
Little suggestion to avoid commenting the yaml